src/App.js

```diff
@@ -1,25 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import Header from "./components/Header";
+import Products from "./components/Products";
+import Cart from "./components/Cart";
```

Comment:
fix: we follow alphabetical import order 
```suggestion
import React, { useState } from "react";

import Cart from "./components/Cart";
import Header from "./components/Header";
import Products from "./components/Products";
```

Reply:
first there are external import followed by internal import seperated by blank line